lib/person_cards.rb

```diff
@@ -23,6 +23,14 @@ def image
     person.image
   end
 
+  def legislature
+    term.legislature
+  end
+
+  def top_identifiers
+    term.top_identifiers
+  end
+
```

Comment:
Sorry for not catching this earlier, but I'm not convinced that these need to be public, do they?